config/routes.js

```diff
@@ -23,6 +23,7 @@ module.exports = function(app) {
   app.get('/absrel', absrel.form);
   app.post('/absrel', absrel.uploadFile);
   app.get('/absrel/:id/original_file/:name', absrel.getMSAFile);
+  app.get('/absrel/:id/fasta', absrel.fasta);
```

Comment:
I feel like we could do something like `app.get('/msa/:id/fasta', msa.fasta);` instead.